src/components/BackOffice/SORContracts/utils.test.js

```diff
@@ -0,0 +1,120 @@
+const {
+  validatePropertyReference,
+  copyContractsSelected,
+  addContractsSelected,
+  dataToRequestObject,
+  propertyReferencesMatch,
+} = require('./utils')
+
+describe('SOR Contracts', () => {
+  describe('validatePropertyReference', () => {
+    it('returns true when valid property reference', () => {
+      // Arrange
+      const propertyReference = '12345678'
+
+      // Act
+      const response = validatePropertyReference(propertyReference)
+
+      // Assert
+      expect(response).toBe(true)
+    })
+
+    it('returns false when invalid property reference', () => {
+      // Arrange
+      const propertyReference = '123456785'
+
+      // Act
+      const response = validatePropertyReference(propertyReference)
+
+      // Assert
+      expect(response).toBe(false)
+    })
+  })
+
+  describe('copyContractsSelected', () => {
+    it('returns true', () => {
```

Comment:
No biggie but I would have probably added a little more other than just 'return true/false' to give a little bit more context on what's being tested (I know it's the function it utils.js). Same applies for lines 45 57 67 74